### PR TITLE
Link to Looker ping explores

### PR DESCRIPTION
**Pull request by wlach**
This is a preliminary implementation and is a bit of a hack. We can
flesh this out later as we deepen our integration with Looker.

### Pull Request checklist

- [x] The pull request has a descriptive title (and a reference to an issue it
      fixes, if applicable)
- [x] All tests and linter checks are passing
- [x] The pull request is free of merge conflicts

### CODE REVIEW

File: src/state/urls.js [hunk 1]
```diff
@@ -16,3 +16,17 @@ export function getBigQueryURL(appName, appId, pingName, metricName) {
 
   return base + (metricName ? `?search=${metricName}` : "");
 }
+
+export function getLookerURL(appName, pingName) {
+  // this is currently only supported for a hardcoded set of app ids and is a bit of a
+  // hack (we should ideally be doing this somewhere in our upstream ETL)
+
+  if (appName === "fenix" || appName === "burnham") {
+    return `https://mozilladev.cloud.looker.com/explore/${appName}/${pingName.replace(
```
Comment (wlach) [inline src/state/urls.js:25]: Should change this when fenix lands on production.

**Review by Iinh — APPROVED** [review]

File: src/state/urls.js (hunk 1 above)
Comment (Iinh) [reply src/state/urls.js:25]: I don't have access to this dev site. is it restricted to only the people who are working on Looker?

Comment (wlach) [reply src/state/urls.js:25]: Yes, though that will almost certainly include you in the near future-- I suspect we should add you to the list of developers for Looker.

Comment (fbertsch) [reply src/state/urls.js:25]: This should be prod now.

**Review by fbertsch — APPROVED** [review]
r+wc
Can we do the same for ios, and the `unified_metrics` explore?

We should be using `namespaces.yaml` for this. We can start with https://bugzilla.mozilla.org/show_bug.cgi?id=1707727, and then move to reading that and using it to know about explores in Looker.

**Comment by wlach** [comment]
>  Can we do the same for ios, and the `unified_metrics` explore?
> 
> We should be using `namespaces.yaml` for this. We can start with https://bugzilla.mozilla.org/show_bug.cgi?id=1707727, and then move to reading that and using it to know about explores in Looker.

Yes to both of these things. Let's see if we can do this the "right way" next week. 😄

**Pull request merged**